source/adapter.js

```diff
@@ -741,7 +741,9 @@ if (navigator.mozGetUserMedia) {
   // the minimum version still supported by adapter.
   webrtcMinimumVersion = 12;
 
-  getUserMedia = navigator.getUserMedia;
```

Comment:
Refer to the JIRA ticket. It seems like Edge doesn't trigger the referenced object

```
function y () { console.log("do something"); }
var x = y;
x(); // opps error
```